module1/src/com/epam/cycles/HomeTask_1_3_3.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.cycles;
+
+public class HomeTask_1_3_3 {
+    public static void main(String[] args) {
+        int sum = 0;
+        for( int i = 1; i <= 100; i++){
+            sum += i * i;
```

Comment:
You can start from 2
```suggestion
        int sum = 1;
        for( int i = 2; i <= 100; i++){
            sum += i * i;
```